ibis/backends/flink/registry.py

```diff
@@ -91,6 +130,16 @@ def _timestamp_from_unix(translator: ExprTranslator, op: ops.Node) -> str:
     return f"TO_TIMESTAMP_LTZ({numeric}, {precision})"
 
 
+def _timestamp_from_ymdhms(
+    translator: ExprTranslator, op: ops.temporal.TimestampFromYMDHMS
+) -> str:
+    year, month, day = op.year.value, op.month.value, op.day.value
+    hours, minutes, seconds = op.hours.value, op.minutes.value, op.seconds.value
```

Comment:
These two lines assume that all of the inputs are literals. Is that a requirement? If not, then you should call `translate` on each one and use the database's string concat functionality to combine the parts.

Reply:
Thanks, fixed the function according to your suggestion, please take a look again.